files/build_templates/docker_image_ctl.j2

```diff
@@ -154,6 +154,10 @@ function postStartAction()
             docker cp $PSENSOR pmon:/usr/bin/
         fi
     fi
+{%- elif docker_container_name == "database-global" %}
+    # Wait until redis starts
+    /usr/bin/docker exec database-global$DEV ping_pong_db_insts supervisor
```

Comment:
Check if swss startup has a ping-pong to validate connectivity

Reply:
swss container startup doesn't have a ping-pong to validate db connection.